Sources/SWBCore/LibSwiftDriver/LibSwiftDriver.swift

```diff
@@ -12,7 +12,7 @@
 
 import Foundation
 
-import SwiftDriver
+public import SwiftDriver
```

Comment:
Ideally we should avoid exposing SwiftDriver types in public API of SWBCore

Reply:
Thanks, fixed that!